File: packages/ts/test/fixtures/fake-validator-repo-ts-only/validator.js
export class FakeTsOnlyValidator {
  name = "fake-ts-only";
  latencyClass = "inline";

  async index(_message, _context) {
    return;
  }

  judge(_message, _context) {
    return { score: 0.5, label: "ts-only" };
  }
}

File: packages/python/tests/fixtures/fake_validator_repo_python_only/validator.py
from galya_reality.base import ContextWindow, Message, ValidationResult, Validator


class FakePythonOnlyValidator(Validator):
    name = "fake-python-only"
    latency_class = "inline"

    async def index(self, message: Message, context: ContextWindow) -> None:
        return None

    def judge(self, message: Message, context: ContextWindow) -> ValidationResult:
        return ValidationResult(score=0.5, label="python-only")
